doc/src/ForManagers.dox

```diff
@@ -55,17 +56,17 @@
  *   example, determining the correct frame range and colorspace of an
  *   image sequence.
  *
- * - The @ref Context supplied to a method can be used to understand
- *   the intentions of the caller, as well as to determine which part of
- *   an application is involved in the call, for example, which image read
- *   node in a compositor is resolving a reference.
+ * - The @ref Context supplied to a method can be used to understand the
+ *   intentions and/or requirements of the caller, as well as to
+ *   determine which part of an application is involved in the call,
+ *   for example, whether the host requires paths or URLs for entities
```

Comment:
[consider] Perhaps choose a different example, given our recent conversation on this